webroot/src/components/Licensee/LicenseeList/LicenseeList.ts

```diff
@@ -174,13 +191,23 @@ class LicenseeList extends Vue {
             }
         }
 
+        // Search params
         requestConfig.compact = this.userStore.currentCompact?.type;
 
-        //
-        // Temp for limited server filtering support
-        //
-        requestConfig.jurisdiction = 'al';
+        if (searchParams?.firstName) {
+            requestConfig.licenseeFirstName = searchParams.firstName;
+        }
+        if (searchParams?.lastName) {
+            requestConfig.licenseeLastName = searchParams.lastName;
+        }
+        if (searchParams?.ssn) {
+            requestConfig.licenseeSsn = searchParams.ssn;
+        }
+        if (searchParams?.state) {
+            requestConfig.jurisdiction = searchParams.state.toLowerCase();
+        }
```

Comment:
Would it be worth testing these new search fields in our LicenseeList test suite?

Reply:
Yep, added.